tests/test_agent.py

```diff
@@ -282,3 +282,11 @@ def test_agentset_shuffle():
     agentset = AgentSet(test_agents, model=model)
     agentset.shuffle(inplace=True)
     assert not all(a1 == a2 for a1, a2 in zip(test_agents, agentset))
+
+def get_Agent():
+    from mesa.agent import Agent
```

Comment:
I don’t think these tests do what you want or expect them to do.

What exactly do you want to test here?

Reply:
The tests kept failing so was updating the necessary files. Mainly circular import was the issue.